test/tests/parallel/set1/spam_virus/amavis.bats

```diff
@@ -2,31 +2,62 @@ load "${REPOSITORY_ROOT}/test/helper/common"
 load "${REPOSITORY_ROOT}/test/helper/setup"
 
 BATS_TEST_NAME_PREFIX='[Amavis + SA] '
-CONTAINER_NAME='dms-test_amavis'
+CONTAINER1_NAME='dms-test_amavis_enabled'
+CONTAINER2_NAME='dms-test_amavis_disabled'
 
 function setup_file() {
-  _init_with_defaults
+  export CONTAINER_NAME
 
+  CONTAINER_NAME=${CONTAINER1_NAME}
+  _init_with_defaults
   local CUSTOM_SETUP_ARGUMENTS=(
     --env ENABLE_AMAVIS=1
     --env AMAVIS_LOGLEVEL=2
     --env ENABLE_SPAMASSASSIN=1
   )
+  _common_container_setup 'CUSTOM_SETUP_ARGUMENTS'
 
+  CONTAINER_NAME=${CONTAINER2_NAME}
+  _init_with_defaults
+   local CUSTOM_SETUP_ARGUMENTS=(
+    --env ENABLE_AMAVIS=0
+    --env ENABLE_SPAMASSASSIN=0
+  )
   _common_container_setup 'CUSTOM_SETUP_ARGUMENTS'
 }
 
 function teardown_file() { _default_teardown ; }
 
-@test 'SpamAssassin integration should be active' {
+@test '(Amavis enabled) configuration should be correct' {
+  export CONTAINER_NAME=${CONTAINER1_NAME}
+
+  _run_in_container grep 'content_filter =' /etc/postfix/main.cf
+  assert_success
+  assert_output --partial "smtp-amavis:[127.0.0.1]:10024"
```

Comment:
```suggestion
  _run_in_container postconf -h content_filter
  assert_success
  assert_line 'smtp-amavis:[127.0.0.1]:10024'
```

I think this will work, although I've chosen `assert_line` as well since the command also outputs:

```
postconf: warning: /etc/postfix/main.cf: unused parameter: dmarc_milter=inet:localhost:8893
postconf: warning: /etc/postfix/main.cf: unused parameter: dkim_milter=inet:localhost:8891
```

Not sure if that's a concern. Alternatively just leave it with the grep approach :+1:

Reply:
Done in 63abdb8824ce2a73f16b66e100f71f0479967065